packages/@react-spectrum/table/src/Table.tsx

```diff
@@ -39,8 +39,8 @@ const DEFAULT_HEADER_HEIGHT = {
 };
 
 const DEFAULT_HIDE_HEADER_CELL_WIDTH = {
-  medium: 34,
-  large: 42
+  medium: 35,
+  large: 43
```

Comment:
adds a px to support the divider

Reply:
What are the implications of this? if there is no divider, is it still ok?

Reply:
Yes
![Screen Shot 2020-10-05 at 4 01 52 PM](https://user-images.githubusercontent.com/30304082/95140685-528b2580-0724-11eb-984f-aca28ed19bf0.png)
![Screen Shot 2020-10-05 at 4 01 30 PM](https://user-images.githubusercontent.com/30304082/95140688-54ed7f80-0724-11eb-9257-171dea56f5ba.png)
![Screen Shot 2020-10-05 at 4 05 36 PM](https://user-images.githubusercontent.com/30304082/95140855-a8f86400-0724-11eb-8a4d-0fb5c99c870e.png)

Reply:
The screenshots don't quite look right, the focus ring around a button shouldn't be clipped like that
I think we still need https://github.com/adobe/react-spectrum/blob/main/packages/%40adobe/spectrum-css-temp/components/table/index.css#L158 and I think we might actually need it to be 
```
.spectrum-Table-cellContents--hideHeader {
  /* allow focus ring of child to extend outside the bounds */
  padding: 4px;
  margin: -4px;
}
```
I suggest this as an alternative because this helps the clipping at the top and bottom

I think it also needs something closer to this:
```
const DEFAULT_HIDE_HEADER_CELL_WIDTH = {
  medium: 36,
  large: 44
};
```
but i don't know where those values were originally pulled from
I still have some clipping on the left with all the above changes, but I missed editing something you've already thought of. Any ideas?

Reply:
So the focus ring around a button is different than a focus ring around textual content check out the table story for crud and focus around the actions `/?path=/story/table--crud`   
I believe that's the expected behavior. So we don't need the negating margin padding combo for this

The width of the button icon was taking into consideration the width of actionButton ie. 
32 and 40 (based on scale). 
The button has a border 1 px so adding 2 px to the overall width 
34 and 42  (based on scale)
 plus the divider 1px
35 and 43  (based on scale)

If we add padding of 4px we will have to increase the width by 8px to accommodate the added pxs